File: java/prime.java
import java.math.BigInteger;
import java.util.*;
class Main {
  public static void main(String[] args) {
    int Count, Counter, Number;
    
    Scanner scan = new Scanner(System.in);
    System.out.print("Enter the number, to find wheather the given number is prime or not:");
    Number = scan.nextInt();
    if(Number <= 3)
    {
      System.out.print(Number + " is a prime number.");
      System.exit(0);
    }
    if(Number % 2 == 0)
    {
      System.out.print(Number + " is not a prime number.");
      System.exit(0);
    }
    Count = 2;
    while(Count * Count <= Number)
    {
      Count++;
    }
    for(Counter = 3; Counter < Count; Counter += 2)
    {
      if(Number % Counter == 0)
      {
        System.out.print(Number + " is not a prime number.");
        System.exit(0);
      }
    }
    System.out.print(Number + " is a prime number.");
    
    
    
  }
}

File: python/prime.py
Number = int(input('Enter the number to find wheather the given number is prime or not:'))
if Number <= 3:
  print str(Number) + " is a prime number."
elif Number % 2 == 0:
  print str(Number) + " is not a prime number."
else:
  Count = 2
  while Count * Count <= Number:
    Count = Count + 1
  Counter = 3
  while Counter <= Count + 1:
    if Number % Counter == 0:
      print str(Number) + " is not a prime number."
      break
    elif Counter == Count:
      Counter = Count + 2
      print str(Number) + " is a prime number."
      break
    Counter = Counter + 1